service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/DraftCMOController.java

```diff
@@ -67,9 +68,9 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToStart(@RequestBody Call
 
     @PostMapping("/mid-event")
     public AboutToStartOrSubmitCallbackResponse handleMidEvent(
-        @RequestBody CallbackRequest callbackRequest,
-        @RequestHeader("authorization") String authorization,
-        @RequestHeader("userId") String userId) {
+        @RequestHeader("authorization") String authorization, @RequestHeader("user-id") String userId,
```

Comment:
According to @Gogetter it throws errors if we leave it in the order it was in before [see here](https://github.com/hmcts/fpl-ccd-configuration/pull/661#discussion_r352807789). The reason the order is actually changed is that in the base branch we made some changes based on a rough idea of what we were going to need but it was never tested because we then decided to do the changes for document generation in a new branch.